client.js

```diff
@@ -174,17 +184,50 @@ class Client {
 
   /**
    * Request events based on the provided query parameters.
+   *
+   * The `query.flags` field uses bitmask values similar to an enum:
+   *   0 = None
+   *   1 = NewestFirst
+   *   2 = TimeRangeBeginExclusive
+   *   4 = TimeRangeEndExclusive
+   *   8 = UseLogStampForTimeRange
+   *
+   * The `query.senderConditions` field can be used to filter by event sender (Source).
+   * The `query.dataConditions` field can be used to filter by data fields (key-value patterns).
+   *
+   * For additional information:
+   *   https://cdpstudio.com/manual/cdp/cdplogger/eventlogreader.html#Flags-enum
+   *   https://cdpstudio.com/manual/cdp/cdplogger/eventlogreader.html#cdp-event-code-flags
+   *
+   * We also support mapping the `evt.code` field to a human-readable string
+   * with `getEventCodeDescription()`.
+   *
    * @param {Object} query - An object matching the EventQuery schema.
-   * For example:
+   * Example:
    * {
    *   timeRangeBegin: 1620000000,
-   *   timeRangeEnd: 1620003600,
-   *   codeMask: 0,
-   *   limit: 100,
-   *   offset: 0,
-   *   flags: 1
+   *   timeRangeEnd:   1620003600,
+   *   codeMask:       0xFFFFFFFF,
+   *   limit:          100,
+   *   offset:         0,
+   *   flags:          1, // e.g. 'NewestFirst'
+   *
+   *   // Example conditions:
+   *   senderConditions: {
+   *     conditions: [
+   *       { value: "*TemperatureSensor*", type: 1 } // 1 = Wildcard
+   *     ]
+   *   },
+   *   dataConditions: {
+   *     pressure: {
```

Comment:
"pressure" - is that from some custom event? Maybe for the example it is better to use a more standard data field, for example the Text column and try to filter maybe the license or suspended alarm events. Then it will be easier for users to test it

![image](https://github.com/user-attachments/assets/bda649ba-a353-4ca7-b541-bdbdf5664345)